src/main/pages/che-7/overview/assembly_hosted-che.adoc

```diff
@@ -1,40 +1,86 @@
+
+image::hosted-che/registration-2.png[]
+
+- Once your environment is being built and then complete, you will receive an email with a link to connect to your Eclipse Che instance.
+
+- Follow the link and the email and start to use Hosted Che by clicking on the *Get Started* button which will redirect you to the User Dashboard.
+
+image::hosted-che/registration-3.png[]
 
+[id="getting-started"]
+== Getting started
+Follow the wizard to create a workspace from the "User Dashboard":
 
-This paragraph is the assembly introduction. It explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. Can include more than one paragraph. Consider using the information from the user story.
+image::hosted-che/getting-started-ud.png[]
 
-[id='prerequisites-{context}']
-== Prerequisites
+You can also create a workspace from a predefined link:https://www.eclipse.org/che/getting-started/cloud/[Getting started]  devfiles which provides a various set of the programming languages and frameworks:
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+- Navigate to https://www.eclipse.org/che/getting-started/cloud/
+- Choose the desired flavor and launch your new Workspace.
 
+image::hosted-che/getting-started-devfile.png[]
 
-Include modules here.
+[id='github-account-linking']
+== Link your GitHub Account
 
+You can also link your account and import projects directly from GitHub during workspace creation from the "User Dashboard":
 
+image::hosted-che/gh-account-linking.png[]
 
-[id='related-information-{context}']
-== Related information
+NOTE: In order to use push the source code to GitHub from IDE you will need to generate SSH keys using the link:https://github.com/eclipse/che-theia/tree/master/plugins/ssh-plugin[SSH Plug-in] and upload the public key to your GitHub account (`Settings` -> `SSH and GPG keys` -> `New SSH key`) 
 
-* A bulleted list of links to other material closely related to the contents of the concept module.
-* For more details on writing assemblies, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
-* Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
+[id='faq']
+== Frequently Asked Questions (FAQs) and Troubleshooting
 
-:context: {parent-context-of-hosted-che}
+Frequently Asked Questions (FAQs) and Troubleshooting can be found on the following link:https://github.com/redhat-developer/rh-che/blob/master/FAQ.adoc[link].
```

Comment:
We are centralizing all the issues/README/wiki to eclipse/che. Wouldn't make sense to move the hosted-che FAQ to eclipse/che as well?

Reply:
good point :+1: